src/extraFiles.qrc

```diff
@@ -3,5 +3,6 @@
         <file>extraFiles/10-usb.network</file>
         <file>extraFiles/g_ether.conf</file>
         <file>extraFiles/usb-ether-gadget.conf</file>
+        <file>extraFiles/rpi-usb-ether-gadget.sh</file>
```

Comment:
Minor nitpick, but if this is being written into the image as `/usr/bin/rpi-usb-ether-gadget` then perhaps this filename should be `extraFiles/rpi-usb-ether-gadget` ? (i.e. drop the `.sh` for consistency?)

Reply:
I added it to the resource with .sh for better detection in editors.